tests/framework/CodeInterfaceTests/CobraTF/gold/CobraTFinterfaceTest4/testDummyStep_MC/1/heat_exchanger.ctf.out

```diff
@@ -0,0 +1,867 @@
+h********************************* input file listing *********************************
+      12345678901234567890123456789012345678901234567890123456789012345678901234567890
+     1       1
+     2        1       0
```

Comment:
You do not need this file, this file is generated by cobra-TF, not the interface.

Reply:
This is the realization of CTF and the variable defined in RAVEN will be extracted from this file. And in the contributing page there is this instruction.

...
Consequentially, the output my_output.out for the first realization must be added in ./raven/tests/framework/CodeInterfaceTests/MyInterfaceTest/MyFirstTest/sampleMyCode/1/ and for the second in ./raven/tests/framework/CodeInterfaceTests/MyInterfaceTest/MyFirstTest/sampleMyCode/2/
…

Shouldn't I keep these?